raiden/blockchain_events_handler.py

```diff
@@ -90,18 +82,20 @@ def handle_tokennetwork_new2(raiden, event, current_block_number):
         token_network_registry_address,
     )
     token_network_proxy = token_network_registry_proxy.token_network(token_network_address)
-
     raiden.blockchain_events.add_token_network_listener(token_network_proxy)
-    token_network_state = create_new_token_network_state(
-        raiden,
-        token_network_proxy,
+
+    token_address = data_decoder(event.event_data['args']['token_address'])
+
+    token_network_state = TokenNetworkState(
+        token_network_address,
+        token_address,
     )
 
-    new_token_network = ContractReceiveNewTokenNetwork(
+    new_payment_network = ContractReceiveNewTokenNetwork(
```

Comment:
the state change name was right

Reply:
I did not intend to change the variable name. Copied stuff from the `handle_tokennetwork_new` to `handle_tokennetwork_new2` in order to make sure the events behaviour is properly duplicated and removed the usage of the now removed `create_new_token_network_state` function. So I copied the name used in the non suffixed function. Will fix.